src/openforms/forms/api/serializers/form.py

```diff
@@ -191,6 +196,14 @@ class Meta:
                 "lookup_url_kwarg": "uuid_or_slug",
             },
         }
+        validators = [
+            AllOrNoneTruthyFieldsValidator(
+                "registration_email_subject",
+                "registration_email_payment_subject",
+                "registration_email_content_html",
+                "registration_email_content_text",
```

Comment:
I went for the same behaviour as for the confirmation email: if you have a custom subject, then you should also have a custom content. Im not sure this is really nice in this case (maybe it's okay to want to only overwrite the payment subject or only the content). :thinking: 

I think it would be nicer to only have:
``` python
AllOrNoneTruthyFieldsValidator("registration_email_content_html", "registration_email_content_text")
```
since the html and text content should have the same info

Reply:
> (maybe it's okay to want to only overwrite the payment subject or only the content). 🤔

I like this. Simple but nice improvement!

However, I value consistent behaviour. Otherwise we get questions about this.
So, I agree by having it this way if we change it for the other emails as well. If not, then we should require both subject and content to be filled.

Reply:
Ok! I will make an issue to change the confirmation emails to have the same behaviour and tackle it in a different PR

Reply:
:+1: I remember struggling with the first time I tried to configure a custom conformation email.